doc/source/how-to/index.rst

```diff
@@ -6,8 +6,103 @@ effective and efficient Python libraries to interface with Ansys products and
 services. Topics also demonstrate how apps and complex services expose
 functionalities such as logging, data transfer, and app APIs.
 
+.. grid:: 3
+
+    .. grid-item-card:: :fas:`fa-regular fa-screwdriver-wrench` Environment setup
+      :link: setting-up
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      How to set up a development environment.
+
+    .. grid-item-card:: :fas:`fa-solid fa-code-compare` Versions
+      :link: supporting-python-versions
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Supporting Python versions.
+
+    .. grid-item-card:: :fas:`fa-solid fa-people-group` Contributing
+      :link: contributing
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Learn how to contribute to PyAnsys.
+
+    .. grid-item-card:: :fas:`fa-solid fa-box-open` Packaging
+      :link: packaging
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      How to package Python libraries.
+
+    .. grid-item-card:: :fas:`fa-solid fa-cubes` gRPC API packages
+      :link: grpc-api-packages
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      How to integrate gRPC in PyAnsys packages.
+
+    .. grid-item-card:: :fas:`fa-solid fa-bars-staggered` Logging
+      :link: logging
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Guidelines for logging in PyAnsys libraries.
+
+    .. grid-item-card:: :fas:`fa-solid fa-file` Documenting
+      :link: documenting
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Guidelines for documentation and style.
+
+    .. grid-item-card:: :fas:`fa-solid fa-flask-vial` Testing
+      :link: testing
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Testing methodology and coverage.
```

Comment:
Change to a how-to description?

How to perform testing and ensure adequate coverage.